include/inexor/vulkan-renderer/camera.hpp

```diff
@@ -1,127 +1,147 @@
 #pragma once
 
 #include <glm/glm.hpp>
+#include <glm/gtc/matrix_transform.hpp>
+
+#include <algorithm>
+#include <array>
+#include <vector>
 
 namespace inexor::vulkan_renderer {
 
-enum class CameraType { LOOKAT, FIRSTPERSON };
+enum class CameraMovement { FORWARD, BACKWARD, LEFT, RIGHT };
+
+// TODO: Implement more camera types.
+enum class CameraType { CAMERA_TYPE_LOOKAT };
 
-/// @brief A RAII wrapper class for camera position and movement.
-/// @todo Refactor method naming!
 class Camera {
```

Comment:
Class is not thread safe.

Reply:
I will add the following warning (just as for KeyboardMouseInput class):

```cpp
/// @warning Not thread safe!
```